manifests/master/windows.pp

```diff
@@ -0,0 +1,24 @@
+# Make sure the windows setup files are on the master for the students
+# to download and configure the Windows Server for the installation of
+# the puppet agent.
+class classroom::master::windows {
+  assert_private('This class should not be called directly')
+
+  File {
+    ensure => file,
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0644',
+  }
+
+  $publicdir   = "/opt/puppetlabs/server/data/packages/public"
```

Comment:
why is this being calculated here instead of using https://github.com/puppetlabs/pltraining-classroom/blob/master/manifests/params.pp#L39